src/vanilla/store.ts

```diff
@@ -84,7 +84,7 @@ type Mounted = {
  * Mutable atom state,
  * tracked for both mounted and unmounted atoms in a store.
  */
-type AtomState<Value = AnyValue> = {
+export type AtomState<Value = AnyValue> = {
```

Comment:
We keep it secret in this file.
```suggestion
type AtomState<Value = AnyValue> = {
```